### PR TITLE
Start to adapt to multiple epochs

### PR DESCRIPTION
**Pull request by Powersource**
For https://github.com/ssbc/ssb-tribes2/pull/65

* [x] can the user add multiple keys to a group? test
* [x] if not, add that functionality
* [x] rename getGroupKeyInfo
* [x] add function for picking write key for a group
* [x] test pick function
* [x] update readme. remember keyring changes bubbling up. pick function. getGroupInfo

### CODE REVIEW

File: format.js [hunk 1]
```diff
@@ -253,7 +253,13 @@ function makeEncryptionFormat() {
     const authorBFE = BFE.encode(authorId)
     const previousBFE = BFE.encode(opts.previous)
 
-    const groupKeys = keyring.group.listSync().map(keyring.group.get)
+    const groupKeys = keyring.group
+      .listSync()
+      .map(keyring.group.get)
+      .map(
+        (groupInfo) => groupInfo.readKeys // readKeys should have format Array<{key, scheme}>
```
Comment (Powersource) [inline format.js:260]: https://github.com/ssbc/envelope-js/blob/8532da86ea84f79ae2532b17e2768586e1c40da6/util/key-slot.js#L31

**Comment by Powersource** [comment]
https://gitlab.com/ahau/lib/ssb-keyring/-/merge_requests/22

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "pull-stream": "^3.6.14",
     "private-group-spec": "^2.1.1",
     "ssb-bfe": "^3.7.0",
-    "ssb-keyring": "^4.1.0",
+    "ssb-keyring": "https://gitlab.com/powersource/ssb-keyring/-/archive/multiple-keys/ssb-keyring-multiple-keys.tar.gz",
```
Comment (Powersource) [inline package.json:21]: https://gitlab.com/ahau/lib/ssb-keyring/-/merge_requests/22

Comment (Powersource) [reply package.json:21]: merged. can someone publish a new major or add me as a maintainer? @arj03 @staltz @mixmix

Comment (arj03) [reply package.json:21]: I tried, but it says forbidden :(

Comment (staltz) [reply package.json]: I made a new major version of keyring now.

**Review by staltz — APPROVED** [review]
Looks great, the test and the readme in particular are nice.

**Pull request merged**